code/modules/mob/living/autohiss.dm

```diff
@@ -96,7 +96,7 @@
 			break
 		. += copytext(message, 1, min_index)
 		if(copytext(message, min_index, min_index+1) == uppertext(min_char))
```

Comment:
```suggestion
		if(copytext(message, min_index, min_index + 1) == uppertext(min_char))
```
I know you didn't touch this part, but can we fix the spacing of this while we are here?